packages/security/src/components/GroupCreate.vue

```diff
@@ -76,20 +79,34 @@
     data () {
       return {
         groupName: '',
+        groupIdentifier: '',
         isCreating: false,
-        isGroupNameAvailable: true,
-        isCheckingGroupName: true
+        groupIdentifierTaken: null
       }
     },
     computed: {
-      groupIdentifier () {
+      groupIdentifierSlug () {
         return slugService.slugify(this.groupName)
+      },
+      invalidGroupIdentifier () {
+        return this.groupIdentifier && !this.groupIdentifier.match(/^[\w-]+$/)
```

Comment:
please add comment describing the regex in human readable form, and unit test it

Reply:
I think this is simple enough as regexes go. Word character or minus. It gets tested in the unit tests with both correct and incorrect group id

Reply:
i agree it's simple enough as regexes go, however the all regex are hard to read as code go's, add a comment seems like a good solution to me.

Reply:
I've added a link to documentation page with examples